tests/prepegMarketOrderBaseAssetAmount.ts

```diff
@@ -270,6 +318,43 @@ describe('prepeg', () => {
 				'base',
 				oraclePriceData
 			);
+
+		const [
+			acquiredBaseReserve,
```

Comment:
these arent being used?

Reply:
correct, just one output, put `_` for now